### PR TITLE
Move to the net461 ref assemblies

### PR DESCRIPTION
**Pull request by jaredpar**
This removes our custom generation in the code and onto the standard net461 reference assemblies.

### CODE REVIEW

File: src/Compilers/Test/Core/TargetFrameworkUtil.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ public enum TargetFramework
         NetStandard20,
         NetCoreApp,
         NetFramework,
-        StandardLatest,
+        NetLatest,
```
Comment (jaredpar) [inline src/Compilers/Test/Core/TargetFrameworkUtil.cs:35]: The reason for this change is everywhere else in the code the term `Standard` meant "subset of real assemblies". As we moved to the real `net461` reference assemblies I moved it to be the full set of assemblies (as it is for all the other core TF). Hence I normalized the names at the same time. 

Eventually I'd like to look at removing the `Standard` variety but that is a longer tail item.

File: src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs [hunk 2]
```diff
@@ -16,6 +16,7 @@
 using System.Reflection.Metadata.Ecma335;
 using System.Text;
 using System.Threading;
+using Basic.Reference.Assemblies;
```
Comment (jaredpar) [inline src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs:19]: This is indeed necessary. The removal of `Net461` from `TestBase` was replaced with the type from the NuPkg and this namespace is required to get to it.

**Comment by jaredpar** [comment]
@dotnet/roslyn-compiler, @jcouv PTAL

File: src/Compilers/Test/Core/TargetFrameworkUtil.cs (hunk 1 above)
Comment (jcouv) [reply src/Compilers/Test/Core/TargetFrameworkUtil.cs:35]: Let's add a comment (this is "latest" on either desktop or Core)

**Comment by jcouv** [comment]
>             Debug.Assert(GetReferences(TargetFramework.NetCoreApp).SequenceEqual(NetCoreApp.References));

Can we add a similar assertion to help for when we add new TargetFramework? Also with a comment

---
Refers to: src/Compilers/Test/Core/TargetFrameworkUtil.cs:181 in 9109edf. [](commit_id = 9109edf6d705df9111b5d8a9bfc47019ce315815, deletion_comment = False)

**Review by jcouv — APPROVED** [review]
LGTM Thanks (iteration 2)

**Pull request merged**